cmd/cmd.go

```diff
@@ -66,8 +66,8 @@ func distroSupportsReExec() bool {
 	if !release.OnClassic {
 		return false
 	}
-	switch release.ReleaseInfo.ID {
-	case "fedora", "centos", "rhel", "opensuse", "suse", "poky", "arch", "manjaro", "lirios", "solus":
+	switch {
+	case release.ReleaseInfo.ID != "debian", !strutil.ListContains(release.ReleaseInfo.IDLike, "debian"):
```

Comment:
Can you please add a similar check for `ID_LIKE=ubuntu`, the code as it stands today would regress on Elementary OS
```
NAME="elementary OS"
VERSION="0.4 Loki"
ID="elementary OS"
ID_LIKE=ubuntu
PRETTY_NAME="elementary OS Loki"
VERSION_ID="0.4"
HOME_URL="http://elementary.io/"
SUPPORT_URL="http://elementary.io/support/"
BUG_REPORT_URL="https://bugs.launchpad.net/elementary/+filebug"
```